crates/cli/src/subcommands/call.rs

```diff
@@ -37,50 +38,36 @@ pub fn cli() -> clap::Command {
         .after_help("Run `spacetime help call` for more detailed information.\n")
 }
 
-pub async fn exec(mut config: Config, args: &ArgMatches) -> Result<(), Error> {
+pub async fn exec(config: Config, args: &ArgMatches) -> Result<(), Error> {
     eprintln!("{}\n", UNSTABLE_WARNING);
-    let database = args.get_one::<String>("database").unwrap();
```

Comment:
What happened to the `database` arg? You're not using it but it's still an argument.

Reply:
We pass `args` to `parse_req()` which extracts it. We do this in a couple other places in the cli, and I figured if I'm reworking the api that I'd move it to the centralized ClientApi struct in the process, so it's easier to see where we're calling from, rather than the http API interaction being spread out all across the crate.